dlt/extract/hints.py

```diff
@@ -351,10 +339,52 @@ def _merge_keys(t_: TResourceHints) -> TPartialTableSchema:
 
         return partial
 
+    @staticmethod
+    def new_table_template(
```

Comment:
looks like you had some serious git conflict? look up for `make_hints`. I removed this one and made it independent function. but it is exactly the same

TLDR;> remove this as use `make_hints`

Reply:
Removed in 7d04fe084b7fbf4a577a7ec6889f7efb61c3cb5b

Apologies! Need to brush up on my gitfu